tests/UtilsTest.php

```diff
@@ -455,4 +457,22 @@ public function testModifyServerRequestQueryParams()
 
         self::assertSame(['name' => 'value'], $modifiedRequest->getQueryParams());
     }
+
+    public function testStreamFor()
+    {
+        $tempResource = Psr7\Utils::tryFopen('php://memory', 'r+');
+        fwrite($tempResource, 'foo');
+
+        $stream = Psr7\Utils::streamFor($tempResource);
+        $this->assertSame('', $stream->read(3));
+
+        fseek($tempResource,0);
```

Comment:
Please fix the code style. Run php-cs-fixer on the repo. This is only done by our CI on master.